test/gtest/uct/test_p2p_err.cc

```diff
@@ -25,7 +25,13 @@ class uct_p2p_err_test : public uct_p2p_test {
         size_t length;
     };
 
-    uct_p2p_err_test() : uct_p2p_test(0) {
+    uct_p2p_err_test() : uct_p2p_test(0, error_handler) {
+    }
+
+    static ucs_status_t
+    error_handler(void *arg, uct_ep_h ep, ucs_status_t status)
```

Comment:
minor: maybe use ucs_empty_function_return_success?

Reply:
ok, will check if it's visible from the test